index.js

```diff
@@ -274,6 +274,10 @@ class RNCallKeep {
   _setupAndroid = async (options) => {
     RNCallKeepModule.setup(options);
 
+    if (options.selfManaged) {
+      return false;
+    }
```

Comment:
If we're running in self managed mode I bail out of this method already.

I realize that this also disables the additionalPermissions being requested, which is a bit opinionated by me. But running CallKeep in self managed mode, would essentially turn it into a permission requesting feature - and there are other better libraries available for that.

But let's discuss! ;)